pkg/custompluginmonitor/custom_plugin_monitor_test.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package custompluginmonitor
+
+import (
+	"testing"
+
+	"k8s.io/node-problem-detector/pkg/problemdaemon"
+)
+
+func TestRegistration(t *testing.T) {
+	if problemdaemon.GetProblemDaemonHandler("custom-plugin-monitor").CmdOptionDescription == "" {
```

Comment:
Checking the cmd option desc feels weird. What if some daemon does not have any customized description? Should we check `problemdaemon.GetProblemDaemonHandler("custom-plugin-monitor")` returns non nil or panic or something like that? This depends on the behavior of `GetProblemDaemonHandler()`.

Same question applies to other daemon registration tests.

Reply:
Done.